src/stringify.ts

```diff
@@ -0,0 +1,34 @@
+import type { SomeSchema } from './types';
+
+export const schemaToString = (schema: SomeSchema<any>): string => {
+  const modifiers = getModifiers(schema);
+  return unionToPrint([schema.toString(), ...modifiers]);
+};
+
+export const typeToPrint = (str: string) => str;
+export const objectToPrint = (str: string) => '{' + str + '}';
```

Comment:
Why do we use `string concatenation` in this particular place and backticks in the other functions?